Permission and Access Helper/force-app/main/default/classes/PermissionServiceImpl.cls

```diff
@@ -0,0 +1,264 @@
+public with sharing class PermissionServiceImpl implements IPermissionService{
+    
+ 	public List<ComboboxDataWrapper> getProfile(){
```

Comment:
format this file by prettier or manually

Reply:
Done